app/views/shops/_fat.html.haml

```diff
@@ -10,7 +10,6 @@
       .trans-sentence
         %div
           %span.fat-taxons{"ng-repeat" => "taxon in hub.taxons"}
-            %render-svg{path: "{{taxon.icon}}"}
```

Comment:
this is removing the icons from the shops page, I think we need a screenshot with before/after so we can see the impact.

Reply:
yes, seeing we don't mess up with margins and padding would be good

Reply:
This PR vs Katuma prod:

**Filters in /shops**

![Screenshot from 2020-07-14 12-05-12](https://user-images.githubusercontent.com/9029026/87413532-b7722c00-c5ca-11ea-82ff-cbed33fce35d.png)
![Screenshot from 2020-07-14 12-05-50](https://user-images.githubusercontent.com/9029026/87413538-b93bef80-c5ca-11ea-973b-b907edd36d0a.png)

**Filters in shop page**

![Screenshot from 2020-07-14 12-07-23](https://user-images.githubusercontent.com/9029026/87413544-bb05b300-c5ca-11ea-91cd-8513f79a66b6.png)
![Screenshot from 2020-07-14 12-07-01](https://user-images.githubusercontent.com/9029026/87413540-ba6d1c80-c5ca-11ea-97a3-b479e448996b.png)

No change, these icons are already missing on most servers.